packages/astro-sst/src/adapter.ts

```diff
@@ -44,9 +40,7 @@ function getAdapter({
         name: baseConfig.name,
         supportedAstroFeatures: {
           ...baseConfig.supportedAstroFeatures,
-          assets: {
```

Comment:
@fwang I'm pretty sure this line is why my build fails due to lack of sharp support. I kept `sharpImageService: "unsupported"` in this MR for parity, but I'm confused: I'm able to deploy on Astro v4 with the current astro plugin. Changing this the value to `stable` fails for a different reason, which makes sense if there actually isn't support.

Do you mind providing some context into sharp support on static sites?

Reply:
Does the Astro build output already have `sharp` bundled in node modules? I vaguely remember it is.
(/cc @nerdoza)

In the context of Lambda, sharp is normally bundled like this https://sst.dev/docs/examples/#sharp-in-lambda

Reply:
Their docs make it seem like it is:
https://docs.astro.build/en/guides/images/#default-image-service

I use pnpm, so I also have sharp added as a dependency. That said, I'm still observing the issue with the passthrough image service. I'll do more testing here to confirm.

As the library owner, would you expect that sharp would be supported with statics sites? It seems like it was, since I was using it, but the configuration in the current adapter code makes it seem like it wasn't/shouldn't. I'm also not deep with Astro adapters, so it's also possible I'm missing something.